docs/downloads/automation-library/integrations/pydoc/review_pydoc_parameters.cm

```diff
@@ -0,0 +1,13 @@
+# -*- mode: yaml -*-
+manifest:
+  version: 1.0
+automations:
+  review_pydoc_parameters: 
+    if:  
+       - {{ source.diff.files | match(attr='diff', regex=r/@param/) | nope }}
```

Comment:
Pydoc uses this convention for method arguments
```
        Args:
            a (float): The first number.
            b (float): The second number.
 ```
 This regex needs to be updated to detect this.